src/SprykerEco/Client/Payone/PayoneClient.php

```diff
@@ -215,4 +217,18 @@ public function sendConsumerScoreRequest(QuoteTransfer $quoteTransfer): Consumer
     {
         return $this->getFactory()->createZedStub()->sendConsumerScoreRequest($quoteTransfer);
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\PayoneKlarnaStartSessionRequestTransfer $klarnaStartSessionRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\PayoneKlarnaSessionResponseTransfer
+     */
+    public function startKlarnaSessionRequest(PayoneKlarnaStartSessionRequestTransfer $klarnaStartSessionRequestTransfer): PayoneKlarnaSessionResponseTransfer
```

Comment:
* Let's rename it to the `sendKlarnaStartSessionRequest()`.

Reply:
++